app/components/EmissionListItem/EmissionListItem.tsx

```diff
@@ -30,38 +33,49 @@ const EmissionListItem: React.FC<EmissionListItemProps> = ({
   title = "",
   co2value = 0,
   onPress,
-}) => (
-  <TouchableOpacity onPress={onPress} style={styles.container}>
-    <View style={styles.iconContainer}>
-      <View style={isMitigated ? styles.mitigatedCircle : styles.notMitigatedCircle} />
+}) => {
+  const useMetricUnits = useSelector(userPreferences.selectors.getUseMetricUnits);
+  const getDisplayUnitsValue = calculation.getDisplayUnitsValue;
+  const getDisplayUnits = calculation.getDisplayUnits;
+
+  const displayValue: number = getDisplayUnitsValue(co2value, useMetricUnits);
+
+  return (
+    <TouchableOpacity onPress={onPress} style={styles.container}>
+      <View style={styles.iconContainer}>
+        <View style={isMitigated ? styles.mitigatedCircle : styles.notMitigatedCircle} />
+        <Ionicons
+          name={iconName as keyof typeof Ionicons.glyphMap}
+          size={22}
+          style={styles.icon}
+          color={isMitigated ? Colors.green50 : Colors.grey70}
+        />
+      </View>
+      <View style={styles.textContainer}>
+        <Text.Primary numberOfLines={1}>{name.length ? name : title}</Text.Primary>
+        <View style={styles.detailsContainer}>
+          <Text.Tertiary numberOfLines={1} lightGray>
+            <FormattedNumber
+              maximumFractionDigits={displayValue >= 1 ? 2 : 4}
+              value={displayValue}
+            />
+            {" " + getDisplayUnits(co2value, useMetricUnits)}CO2
```

Comment:
Just realized that now : we are missing the `eq` for some reason 😅 `{" " + getDisplayUnits(co2value, useMetricUnits)}CO2eq`